drivers/serial/uart_nrfx_uart.c

```diff
@@ -996,7 +996,9 @@ static int uart_nrfx_init(const struct device *dev)
 	nrf_gpio_cfg_output(TX_PIN);
 
 	if (RX_PIN_USED) {
-		nrf_gpio_cfg_input(RX_PIN, NRF_GPIO_PIN_PULLUP);
+		nrf_gpio_cfg_input(RX_PIN,
+			HAS_PROP(rx_pull_up) ? NRF_GPIO_PIN_PULLUP
```

Comment:
I've got a second look and propably this HAS_PROP needs to be replaced by PROP.

Reply:
Right. Thanks for catching this. I'll fix it promptly.